example/centos/centos-9-x86_64-qcow2.yaml

```diff
@@ -6,97 +6,131 @@ otk.define:
     # empty
   kernel:
     cmdline: console=tty0 console=ttyS0,115200n8 no_timer_check net.ifnames=0
+  packages:
+    build:
+      otk.external.osbuild-gen-depsolve-dnf4:
+        architecture: x86_64
+        module_platform_id: f40
+        releasever: "40"
+        repositories:
+          - id: "AppStream"
+            baseurl: "https://rpmrepo.osbuild.org/v2/mirror/public/el9/cs9-x86_64-appstream-20240901"
+          - id: "BaseOS"
+            baseurl: "https://rpmrepo.osbuild.org/v2/mirror/public/el9/cs9-x86_64-baseos-20240901"
+        packages:
+          include:
+            - selinux-policy-targeted
+            - coreutils
+            - xz
+            - glibc
+            - platform-python
+            - systemd
+            - python3
+            - grub2-pc
+            - xfsprogs
+            - dosfstools
+            - rpm
+            - policycoreutils
+            - selinux-policy-targeted
+            - grub2-pc
+            - xfsprogs
+            - dosfstools
+            - rpm
+            - policycoreutils
+            - selinux-policy-targeted
+            - qemu-img
+          exclude: []
+    os:
+      otk.external.osbuild-gen-depsolve-dnf4:
+        architecture: x86_64
+        module_platform_id: f40
+        releasever: "40"
+        repositories:
+          - id: "AppStream"
+            baseurl: "https://rpmrepo.osbuild.org/v2/mirror/public/el9/cs9-x86_64-appstream-20240901"
+          - id: "BaseOS"
+            baseurl: "https://rpmrepo.osbuild.org/v2/mirror/public/el9/cs9-x86_64-baseos-20240901"
+        packages:
+          include:
+            - dracut-config-generic
+            - grub2-pc
+            - dracut-config-generic
+            - efibootmgr
+            - grub2-efi-x64
+            - shim-x64
+            - kernel
+            - xfsprogs
+            - dosfstools
+            - selinux-policy-targeted
+            - "@core"
+            - authselect-compat
+            - chrony
+            - cloud-init
+            - cloud-utils-growpart
+            - cockpit-system
+            - cockpit-ws
+            - dnf-utils
+            - dosfstools
+            - nfs-utils
+            - oddjob
+            - oddjob-mkhomedir
+            - psmisc
+            - python3-jsonschema
+            - qemu-guest-agent
+            - redhat-release
+            - redhat-release-eula
+            - rsync
+            - tar
+            - tuned
+            - tcpdump
+          exclude:
```

Comment:
Finding a way to test this would be really good, this will probably require changes in the `gen-manifests` mock dnfjson mode in images though so probably followup material but IMHO quite important to ensure we really have the same images.